### PR TITLE
PR Template: edge cases

### PR DESCRIPTION
**Pull request by fm3**
I suggest adapting the PR Template like this
 - Remove “ready for review“ checkbox – Github’s Draft PR feature supersedes that. Also we usually write “WIP:” in the PR title. No need for another repetition.
 - Add a new list of edge cases to consider when making big changes / new features. I started with a short list, feel free to add yours! I guess we shouldn’t have 100 entries, but maybe a few common things that we missed in the past. What do you think?

(I requested many reviews to give you all the chance to chime in, but don’t feel obligated)

### CODE REVIEW

**Review by normanrz — APPROVED** [review]

File: .github/common_edge_cases.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Common edge cases to consider when making significant changes to WEBKNOSSOS.
+
+Will this also work if
+ - Mag1 does not exist
+ - There are multiple organizations
+ - There is only one organization
+ - The user is Team Manager, but not Admin/Dataset Manager
+ - The annotation is a compound task/project/tasktype annotation
+ -
```
Comment (philippotto) [inline .github/common_edge_cases.md:9]:
```suggestion
Will this also work if
 - Mag1 does not exist
 - There are multiple organizations
 - There is only one organization
 - The user is Team Manager, but not Admin/Dataset Manager
 - The annotation is a compound task/project/tasktype annotation
 - The user is logged out and views a public dataset or annotation
```

File: .github/common_edge_cases.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Common edge cases to consider when making significant changes to WEBKNOSSOS.
+
+Will this also work if
+ - Mag1 does not exist
+ - There are multiple organizations
+ - There is only one organization
+ - The user is Team Manager, but not Admin/Dataset Manager
+ - The annotation is a compound task/project/tasktype annotation
```
Comment (philippotto) [inline .github/common_edge_cases.md:8]:
```suggestion
 - The annotation is a compound task/project/tasktype annotation
 - The user is logged out and views a public dataset or annotation
```

**Review by philippotto — APPROVED** [review]
Good ideas :+1:

**Review by frcroth — COMMENTED** [review]
Perhaps add something like "Removed development-only changes to application.conf" or something more generic? I forgot that in the last PR

**Pull request merged**